src/demo/app.component.html

```diff
@@ -22,6 +22,9 @@
         <li>
           <a routerLink="/emptystate" routerLinkActive="active">Empty State</a>
         </li>
+        <li>
+          <a routerLink="/pagination" routerLinkActive="active">Pagination</a>
```

Comment:
Please move the 'Pagination' link so it appears alphabetically in the example list